cmd/solo-kit-gen/main.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+	"github.com/solo-io/solo-kit/pkg/code-generator/codegen"
+	"github.com/solo-io/solo-kit/pkg/code-generator/docgen"
+	"github.com/solo-io/solo-kit/pkg/code-generator/model"
+	"github.com/solo-io/solo-kit/pkg/code-generator/parser"
+	"github.com/solo-io/solo-kit/pkg/errors"
+	"github.com/solo-io/solo-kit/pkg/utils/log"
+)
+
+func Gopath() string {
+	return os.Getenv("GOPATH")
+}
+
+var commonImports = []string{
+	"-I" + Gopath() + "/src",
```

Comment:
I thought the parameter syntax was "-I=...", so it seems like Gopath() is expected to be "=...", which would be weird. Is this missing a "="?

Reply:
the `=` is optional